setup.py

```diff
@@ -67,10 +67,11 @@
         'Programming Language :: Python :: 3.2',
         'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.8',
```

Comment:
Might be the right time to drop the py2 support too.

May be it could be done through a follow up patch.
Thoughts?

Reply:
Good idea, I'll get onto it. what should I do about the package version though?
I'm not familiar with this syntax.

Reply:
You just need to drop the backward compatibility by removing py27 specific syntax in the code by example by replacing `raw_input` by `input`.

When you'll substitute all these specific then you just have to drop the lines with `Python :: 2*` (2.7, 2).
After all your substitutions you should ensure that the code work as expected with py3. Maybe by using tox or virtualenv.

Let me know if I reply to your questions.